### PR TITLE
cel-spec@0.14.0

### PR DESCRIPTION
**Pull request by mmorel-35**
Release: https://github.com/google/cel-spec/releases/tag/v0.14.0

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (cel-spec) have been updated in this PR. Please review the changes.

File: modules/cel-spec/metadata.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "homepage": "https://github.com/google/cel-spec",
+    "maintainers": [
+        {
+            "email": "bcr-maintainers@bazel.build",
```
Comment (Wyverald) [inline modules/cel-spec/metadata.json:5]: mind adding yourself as a maintainer here?

Comment (Wyverald) [reply modules/cel-spec/metadata.json:5]: fair enough :)

**Review by meteorcloudy — APPROVED** [review]
Looks good, but I strongly recommend to upstream the patches if possible so that we don't need them for future versions.

File: modules/cel-spec/0.14.0/patches/extensions_dot_bzl.patch [hunk 2]
```diff
@@ -0,0 +1,21 @@
+diff --git a/extensions.bzl b/extensions.bzl
+new file mode 100644
+index 0000000..c7e96b3
+--- /dev/null
++++ b/extensions.bzl
+@@ -0,0 +1,15 @@
++load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
++
++def _non_module_dependencies_impl(_ctx):
++    http_archive(
```
Comment (meteorcloudy) [inline modules/cel-spec/0.14.0/patches/extensions_dot_bzl.patch:10]: Have you tried to turn googleapis into a proper Bazel module?

Comment (meteorcloudy) [reply modules/cel-spec/0.14.0/patches/extensions_dot_bzl.patch:10]: Thanks!

**Pull request merged**